kubernetes/core/src/main/resources/META-INF/core.xml

```diff
@@ -18,6 +18,9 @@
 <idea-plugin>
     <extensions defaultExtensionNs="com.intellij">
         <applicationService serviceImplementation="com.google.kubernetes.tools.core.PluginInfo"/>
+        <applicationService serviceImplementation="com.google.container.tools.core.settings.KubernetesSettingsService"/>
 
+        <!--<applicationConfigurable parentId="google.settings" id="google.kubernetes.settings"-->
```

Comment:
commented out to hide the new menu item until this configurable path is consumed in the next PR